src/client/Microsoft.Identity.Client/LegacyManagedIdentity/AbstractManagedIdentity.cs

```diff
@@ -43,39 +42,63 @@ protected AbstractManagedIdentity(RequestContext requestContext, ManagedIdentity
                 cancellationToken.ThrowIfCancellationRequested();
             }
 
+            HttpResponse response = null;
+
             // Convert the scopes to a resource string.
             string resource = parameters.Resource;
 
             ManagedIdentityRequest request = CreateRequest(resource);
 
+            _requestContext.Logger.Info("[Managed Identity] sending request to managed identity endpoints.");
+
             try
             {
-                HttpResponse response =
-                    request.Method == HttpMethod.Get ?
-                    await _requestContext.ServiceBundle.HttpManager
-                        .SendGetForceResponseAsync(
-                            request.ComputeUri(), 
-                            request.Headers, 
-                            _requestContext.Logger, 
-                            cancellationToken: cancellationToken).ConfigureAwait(false) :
-                    await _requestContext.ServiceBundle.HttpManager
-                        .SendPostForceResponseAsync(
-                            request.ComputeUri(), 
-                            request.Headers, 
-                            request.BodyParameters, 
-                            _requestContext.Logger, cancellationToken: cancellationToken).ConfigureAwait(false);
-
-                return await HandleResponseAsync(parameters, response, cancellationToken).ConfigureAwait(false);
+                if (request.Method == HttpMethod.Get)
+                {
+                    response = await _requestContext.ServiceBundle.HttpManager
+                        .SendRequestAsync(
+                            request.ComputeUri(),
+                            request.Headers,
+                            body: null,
+                            HttpMethod.Get,
+                            logger: _requestContext.Logger,
+                            doNotThrow: true,
+                            retry: true,
+                            mtlsCertificate: null,
+                            cancellationToken).ConfigureAwait(false);
+                }
+                else
+                {
+                    
+                    Debug.WriteLine("_sourceType in AuthenticateAsync" + _sourceType);
```

Comment:
need to remove

Reply:
Whatever, the compiler will remove it from the final product....